out/js/credit_card_form.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+var CreditCardForm = (function ($) {
+
+  var debug = true;
+
+  function callback(response) {
+    $("#creditcard-form-div").height(350).fadeIn();
+    getOrderButton().prop("disabled", false);
+
+    if (debug) {
+      console.log(response);
+    }
+  }
+
+  function logError(where, error) {
+    if (typeof error == "string") {
+      $("#wirecard-cc-error").html(error)
+    }
+
+    if (debug) {
+      console.error("Error on " + where + ":", error);
+    }
+  }
+
+  function getOrderButton() {
+    return $('#orderConfirmAgbBottom button[type = "submit"]');
+  }
+
+  function submitPaymentForm(event) {
+    if (!$("#wirecard-cc-form input#jsresponse").length) {
+      event.preventDefault();
+      WirecardPaymentPage.seamlessSubmitForm({
+        onSuccess: setParentTransactionId,
+        onError: function (error) {
+          logError('seamlessSubmitForm', error);
+        }
+      })
+    }
+  }
+
+  function setParentTransactionId(response) {
+    console.log("setParentTransactionId", response);
+    var form = $('#wirecard-cc-form');
+    $.each(response, function (key, value) {
+      form.append("<input type='hidden' name='" + key + "' value='" + value + "'>");
+    });
+    form.append("<input type='hidden' id='jsresponse' name='jsresponse' value='true'>");
+    form.submit();
+  }
+
+  return obj = {
```

Comment:
```suggestion
  return {
```

`obj` is not used anywhere.

Reply:
👍